### DISCUSSION TITLE
Inquiry about Elasticsearch version compatibility with SkyWalking

### DISCUSSION THREAD

**Discussion by xy-zzs** [apache/skywalking#11069] [Q&A] [answered]
I am using SkyWalking for application performance monitoring and I would like to know the recommended Elasticsearch version for use with SkyWalking. I have checked the official documentation but couldn't find specific information regarding the Elasticsearch version compatibility.

Could you please provide guidance on the Elasticsearch version that is recommended to work with SkyWalking v9.3.0? If there are any known compatibility issues or considerations, please let me know as well.

Thank you for your assistance!

**Comment by wu-sheng** [comment, marked as answer]
It is on the `choose storage` docs. Every version has its doc, you could check from website.

**Reply by xy-zzs** [reply]
OK, I found it. thanks
